modules/rosa-additional-principals-overview.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-additional-principals-overview_{context}"]
+= Additional principals on your ROSA cluster
+
+You can allow AWS IAM Roles as additional principals to connect to your cluster's private API server endpoint.
+
+You can access your {hcp-title} cluster's API Server endpoint from either the public internet or the interface endpoint that was created within the VPC private subnets. By default, you can privately access your {hcp-title} API Server by using the `-kube-system-kube-controller-manager` Operator role. To be able to access ROSA with HCP API server from another account directly without using the primary account where cluster is installed, you must include cross-account IAM Roles as additional principals. This feature allows you to simplify your network architecture and reduce data transfer costs by avoiding peering or attaching cross-account VPCs to cluster's VPC.
+
+image::AWS_cross_account_access.png[Overview of AWS cross account access]
+
+In this diagram, the cluster creating account is designated as Account A. This account designates that another account, Account B, should have access to the API server.
```

Comment:
_suggesting to include following note_

Once configuring additional allowed principals, you will need to create Interface VPC Endpoint in the VPC from where you want to access the cross-account ROSA with HCP API Server. Then, create private hosted zone in Route53 to route calls made to cross-account ROSA with HCP API Server to pass through the created VPC Endpoint.